plugins/node/instrumentation-dataloader/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@opentelemetry/instrumentation-dataloader",
+  "version": "0.1.0",
+  "description": "OpenTelemetry instrumentation for dataloader",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "tsc -p .",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/instrumentation-dataloader --include-dependencies",
+    "prewatch": "npm run precompile",
+    "prepare": "npm run compile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test-all-versions": "tav",
+    "version:update": "node ../../../scripts/version-update.js"
+  },
+  "keywords": [
+    "dataloader",
+    "instrumentation",
+    "nodejs",
+    "opentelemetry",
+    "profiling",
+    "tracing"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.12.0"
```

Comment:
We technically don't support node 12 and lower any more. Please match this with other packages.